src/middlewares/githubauth2.middleware.js

```diff
@@ -0,0 +1,14 @@
+/* eslint-disable import/prefer-default-export */
+import { check } from 'express-validator/check';
```

Comment:
If there is already a githubAuth middleware, these functions can be placed inside it and the files can be combined